File: codeforces/282A/cpp/282a.cpp
#include <iostream>
#include <string>
using namespace std;

int main()
{
    int n, result = 0;
    cin >> n;

    for (int i = 0; i < n; i++) {
        string cmd;
        cin >> cmd;
        if (cmd == "++X" || cmd == "X++") {
            result++;
        }
        else if (cmd == "--X" || cmd == "X--") {
            result--;
        }
    }
    cout << result << endl;

    return 0;
}

File: codeforces/282A/go/282a.go
package main

import "fmt"

func main() {
    var n int
    var result int = 0
    var cmd string

    fmt.Scan(&n)
    for i := 0; i < n; i++ {
        fmt.Scan(&cmd)
        if cmd == "++X" || cmd == "X++" {
            result++
        } else if cmd == "--X" || cmd == "X--" {
            result--
        }
    }
    fmt.Println(result)
}
